sdk/go/azure/compute/orchestratedVirtualMachineScaleSet.go

```diff
@@ -71,7 +69,10 @@ type OrchestratedVirtualMachineScaleSet struct {
 	DataDisks               OrchestratedVirtualMachineScaleSetDataDiskArrayOutput           `pulumi:"dataDisks"`
 	EncryptionAtHostEnabled pulumi.BoolPtrOutput                                            `pulumi:"encryptionAtHostEnabled"`
 	EvictionPolicy          pulumi.StringPtrOutput                                          `pulumi:"evictionPolicy"`
-	Identity                OrchestratedVirtualMachineScaleSetIdentityPtrOutput             `pulumi:"identity"`
+	Extensions              OrchestratedVirtualMachineScaleSetExtensionArrayOutput          `pulumi:"extensions"`
+	// Specifies the time alloted for all extensions to start. The time duration should be between 15 minutes and 120 minutes (inclusive) and should be specified in ISO 8601 format. The default value is 90 minutes (PT1H30M).
```

Comment:
s/alloted/allotted

Reply:
This is an upstream comment we have no real control over